format/spec.md

```diff
@@ -1117,27 +1136,28 @@ Schemas are serialized as a JSON object with the same fields as a struct in the
+| **`map`**                                   | `JSON object: {`<br />&nbsp;&nbsp;`"type": "map",`<br />&nbsp;&nbsp;`"key-id": <key id int>,`<br />&nbsp;&nbsp;`"key": <type JSON>,`<br />&nbsp;&nbsp;`"value-id": <val id int>,`<br />&nbsp;&nbsp;`"value-required": <bool>`<br />&nbsp;&nbsp;`"value": <type JSON>`<br />`}`                                                                                                                                                                                                                                                                                                                 | `{`<br />&nbsp;&nbsp;`"type": "map",`<br />&nbsp;&nbsp;`"key-id": 4,`<br />&nbsp;&nbsp;`"key": "string",`<br />&nbsp;&nbsp;`"value-id": 5,`<br />&nbsp;&nbsp;`"value-required": false,`<br />&nbsp;&nbsp;`"value": "double"`<br />`}`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| **`geometry(C, E)`**                        | `JSON object: {`<br />&nbsp;&nbsp;`"type": "geometry",`<br />&nbsp;&nbsp;`"crs": <C>,`<br />&nbsp;&nbsp;`"edges": <E>`<br />`}`                                                                                                                                                                                                                                                                                                                                                                                                                       | `{`<br />&nbsp;&nbsp;`"type": "geometry",`<br />&nbsp;&nbsp;`"crs": "my.crs.table.property",`<br />&nbsp;&nbsp;`"edges": "planar"`<br />`}`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
I think the example should be a well-known CRS, like `OGC:CRS84`. We should also add a _note_ that states that implementations are encouraged to store the PROJJSON representation of the CRS in a table property using the ID from this field.